workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -692,6 +692,19 @@ pre[class*='language-'] {
 	}
 }
 
+.search-banner {
+	&::after {
+		top: 155px !important;
+	}
+
+	&::before {
+		top: 198px !important;
+	}
+
+	height: 198px !important;
```

Comment:
properties should go above more nesting, so these should be moved above the before and after

Reply:
Done!